marginesy.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <link rel="stylesheet" href="akapit.css">
+    <meta charset="UTF-8">
+    <meta charset="UTF-8">
+    <title>Marginesy</title>
+</head>
+<body>
+
+<h1>Marginesy</h1>
+
+<h2>Akapit tekstu</h2>
+<p>
+    Aliquam non ut hic ea et quam. <span>Dolores esse quis totam</span> aut et consequatur quia hic iste commodi quis
+    libero explicabo. Natus ut labore sit <span>libero animi consequuntur</span> nulla.
+</p>
+<p class="space">
+    Dolor architecto sed consequatur incidunt ullam ut quis et culpa nihil soluta. Dolores maxime officiis molestiae
+    consectetur hic eum est sapiente ullam sunt. Nam qui eum temporibus iusto repudiandae ipsum iste sint nemo ut. Et
+    mollitia officiis vitae rem ullam nihil quae quas eos et.
+</p>
+<p>
+    Beatae error eos et ut iusto. Quae unde et et et beatae magnam iure cumque voluptatem est molestiae rerum quam
+    sunt.
+</p>
+
+<h2>Boxy z tekstem</h2>
+<div style="display: flex; gap: 10px">
+    <div class="gap" style="flex: 4 0">
+        Voluptas est et et distinctio voluptate nihil unde est. Nemo aperiam suscipit iste qui libero voluptas ex
+        dolor nihil animi enim quibusdam. Veniam sapiente facilis et sequi atque vitae quos provident ipsam in dolor.
+        Omnis reiciendis sint excepturi voluptatem incidunt et et sunt et.
+    </div>
+    <div class="delete" style="flex: 4 0">
+        Numquam sint doloribus perferendis aspernatur illo maxime beatae eum nisi et possimus. Placeat culpa non et
+        vel sit veniam qui architecto dolor nostrum laudantium dicta ut quis.
+    </div>
+    <div style="flex: 2 0">
+        Numquam sint doloribus perferendis aspernatur illo maxime beatae eum nisi et possimus.
+    </div>
+</div>
+
+<h2>8 kolorów</h2>
+<div style="display: flex;">
+    <div class="box" style="width: 30px; height: 30px; border: 1px dotted darkgray"></div>
+    <div class="box" style="width: 30px; height: 30px; border: 1px dotted darkgray"></div>
+    <div class="box" style="width: 30px; height: 30px; border: 1px dotted darkgray"></div>
+    <div class="box" style="width: 30px; height: 30px; border: 1px dotted darkgray"></div>
+    <div class="box" style="width: 30px; height: 30px; border: 1px dotted darkgray"></div>
+    <div class="box" style="width: 30px; height: 30px; border: 1px dotted darkgray"></div>
+    <div class="box" style="width: 30px; height: 30px; border: 1px dotted darkgray"></div>
+    <div class="box" style="width: 30px; height: 30px; border: 1px dotted darkgray"></div>
```

Comment:
Dobrze byloby, aby ostatni `.box` nie mial marginesu z prawej strony 👇
<img width="1436" height="1218" alt="Screenshot 2025-11-27 at 10 23 21" src="https://github.com/user-attachments/assets/537fc5b3-3ab7-43e6-82ee-09bf6f1bd6de" />
Jak to mozesz zrobic? Podpowiedz: pseudoklasy. Tego w zadaniu nie bylo, ale chcialbym abys to przecwiczyl. W razie potrzeby pomoge.